### PR TITLE
Shorten the query genes in Enrichments box plot x-axis

### PR DESCRIPTION
**Pull request by ersinciftci**
Fixes https://github.com/cBioPortal/cbioportal/issues/4324

### CODE REVIEW

File: src/pages/resultsView/enrichments/MiniBoxPlot.tsx [hunk 1]
```diff
@@ -137,9 +137,9 @@ export default class MiniBoxPlot extends React.Component<IMiniBoxPlotProps, {}>
                                 </div>
                                 <VictoryChart domainPadding={{ x: 60, y: 20 }} containerComponent={<VictoryContainer responsive={false} 
                                     containerRef={(ref: any) => this.svgContainer = ref}/>} height={350} width={350} 
-                                    padding={{ top: 50, bottom: 60, left: 60, right: 40 }} theme={CBIOPORTAL_VICTORY_THEME}>
+                                    padding={{ top: 50, bottom: 70, left: 60, right: 40 }} theme={CBIOPORTAL_VICTORY_THEME}>
                                     <VictoryAxis tickValues={[1, 2]} tickFormat={(t: any) => t === 1 ? "Altered" : "Unaltered"}
-                                        label={"Query: " + shortenGenesLabel(this.props.queryGenes, 3) + " (q-Value: " + 
+                                        label={"Query: " + shortenGenesLabel(this.props.queryGenes, 2) + "\n(q-Value: " +
```
Comment (alisman) [inline src/pages/resultsView/enrichments/MiniBoxPlot.tsx]: i'm surprised that a slash n has any effect.  i think @adamabeshouse discovered a way of forcing wrapping in a Victory label.  Maybe passing it an array of strings?

Comment (alisman) [reply src/pages/resultsView/enrichments/MiniBoxPlot.tsx]: is that what you were trying to accomplish?

Comment (ersinciftci) [reply src/pages/resultsView/enrichments/MiniBoxPlot.tsx]: According to the docs, newline character and sending an array has the same effect https://formidable.com/open-source/victory/docs/victory-label/#text

Comment (adamabeshouse) [reply src/pages/resultsView/enrichments/MiniBoxPlot.tsx]: @alisman this is the same approach I ended up taking.. according to a gitter reply from boygirl in march, this is the only thing thats possible, theres no automatic line wrap

Comment (alisman) [reply src/pages/resultsView/enrichments/MiniBoxPlot.tsx]: gotcha

**Pull request merged**